AndroidSDK/build.gradle

```diff
@@ -13,9 +13,9 @@ allprojects {
 dependencies {
     // Compile dependencies will be added as dependency in pom file.
     //noinspection GradleDynamicVersion
-    compile 'com.android.support:support-v4:[14.0.0,26.+['
+    compile 'com.android.support:support-v4:[14.0.0,)'
```

Comment:
This seems to fetch alpha version again:
`gradle dependencies`
`+--- com.android.support:support-v4:[14.0.0,) -> 26.0.0-alpha1`
Is this intended?

remember: https://stackoverflow.com/questions/43527020/gradle-not-respecting-ivy-standard-version-requirement/43561657#43561657 ?